shuffle/cli/src/shared.rs

```diff
@@ -25,8 +25,15 @@ use url::Url;
 
 pub const MAIN_PKG_PATH: &str = "main";
 const NEW_KEY_FILE_CONTENT: &[u8] = include_bytes!("../new_account.key");
-pub const LOCALHOST_NETWORK_NAME: &str = "localhost";
-pub const LOCALHOST_NETWORK_BASE: &str = "http://127.0.0.1";
+
+pub const LOCALHOST_NAME: &str = "localhost";
+pub const LOCALHOST_JSON_RPC_URL: &str = "http://127.0.0.1:8080";
+pub const LOCALHOST_DEV_API_URL: &str = "http://127.0.0.1:8081";
+
+pub const TROVE_TESTNET_NETWORK_NAME: &str = "trove";
+pub const TROVE_TESTNET_JSON_RPC_URL: &str = "http://trove.aws.hlw3truzy4ls.com/";
+pub const TROVE_TESTNET_DEV_API_URL: &str = "https://api.trove.aws.hlw3truzy4ls.com/";
+pub const TROVE_TESTNET_FAUCET_URL: &str = "https://trove.aws.hlw3truzy4ls.com/mint";
```

Comment:
We want the base urls for these endpoints, because we will build the desired REST endpoint from that. ie:
`POST https://base.com/accounts`.
1. Remove trailing `/mint`
2. Validate that you only have a base url

Reply:
Was a bit confused here. Used the faucet_endpoint from https://fb.quip.com/UytvAaPdWsu4. Using new faucet endpoint for account creation found here: https://github.com/diem/diem/pull/9568/files#diff-841273dacd152ce5926f77cdf78ac63171ba37b969726f5d4bfb63f5bf6ca060R123